numpy/_core/_add_newdocs.py

```diff
@@ -931,6 +931,14 @@
         'A' (any) means 'F' if `a` is Fortran contiguous, 'C' otherwise
         'K' (keep) preserve input order
         Defaults to 'K'.
+    device : str, optional
+        The device on which to place the created array. Default: None.
+
+        .. note::
```

Comment:
I realize there has been discussion about this, but for people looking at docstrings in `ipython`, all this extra space taken by something that cannot realistically be used seems a bit of a waste.  Does `sphinx` allow the following?
```
The device on which to place the created array. Default: None.
.. note:: Only the ``"cpu"`` device is supported by NumPy.
```
If not, I'd suggest to not worry about the pdf'd docs and just write:
```
The device on which to place the created array. Default: None.
Note that only the ``"cpu"`` device is supported by NumPy.
```

Reply:
Agreed, how about something:  For Array-API interoperability only, so must be ``"cpu"`` if passed.  See also `device keyword`_  (some link into the glossary maybe)?

(Fine without the link also, I suppose if we drop the "must be cpu if given" and only have a link, it might fit on a single row, in which case, maybe...)

Reply:
Good to keep it compact indeed - just the sentence LGTM.